telos-frontend/src/views/journal/Page.jsx

```diff
@@ -1,44 +1,27 @@
-import { useState, useCallback, useEffect } from 'react';
+import { useCallback } from 'react';
 import { useDrop } from 'react-dnd';
 import update from 'immutability-helper';
 import DraggableWidget from '../../dnd/DraggableWidget';
 import { WidgetTypes } from '../../dnd/WidgetTypes';
 import pageStyles from './Page.module.css';
 
-const Page = ({ date, leftPage }) => {
-  const [widgets, setWidgets] = useState({});
-
-  useEffect(() => {
-    // Temp code to display all widgets. Remove when backend is ready.
-    if (leftPage) {
-      setWidgets({
-        0: { widgetType: 'todo', top: 0, left: 0 },
-        1: { widgetType: 'habit_tracker', top: 200, left: 200 },
-      });
-    } else {
-      setWidgets({
-        2: { widgetType: 'text', top: 100, left: 100 },
-        3: { widgetType: 'calendar', top: 300, left: 300 },
-      });
-    }
-
-    // TODO: Fetch widgets
-  }, [date]);
-
+// eslint has been disabled here as the unused args date and leftPage might be used in future.
+// eslint-disable-next-line no-unused-vars
+const Page = ({ date, leftPage, activeWidgets, setActiveWidgets }) => {
```

Comment:
leftPage prop is no longer required. Can this be removed?

Can you also rename activeWidgets to widgets so it makes more sense? Each page either has widgets or it doesn't, so "active" doesn't really apply here.

Reply:
All done.